packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -289,9 +289,14 @@ contract VotingV2 is
     }
 
     function _updateTrackers(address voterAddress) internal override {
+        // todo: the ordering of these functions is critical and should be though about more deeply + comments.
         _updateCumulativeSlashingTrackers();
         _updateAccountSlashingTrackers(voterAddress);
-        _updateReward(voterAddress);
+        super._updateTrackers(voterAddress);
+    }
+
+    function inActiveReveal() internal override returns (bool) {
+        return (pendingPriceRequests.length > 0 && getVotePhase() == Phase.Reveal);
```

Comment:
Does this still work if we're in a reveal phase that has no active votes, but some votes are queued for the _next round_?

Reply:
ye, you right. this will block this...I'll think on a solution.

Reply:
I swapped this to `getPendingRequests()` which only returns the requests active in this current round. This _is_ more expensive but it'll give us what we need for now and we can optimize it a bit later.